### ISSUE TITLE
Generated client for github is broken

### ISSUE THREAD

**Issue by CGQAQ** [oxidecomputer/progenitor#394]
https://github.com/oxidecomputer/third-party-api-clients/blob/abc7b86ac3dbb444736192c502af04e81e3db5a9/github/src/types.rs#L257

![image](https://user-images.githubusercontent.com/15936231/228107085-57bf175c-670f-42a8-bdf1-58aab8fabbac.png)

https://github.com/serde-rs/serde/issues/2415

there's a TON of `#[serde()]`, and has been broke since serde 1.0.157.

pin: serde-rs/serde/pull/2421

**Comment by CGQAQ** [comment]
fixed in the upstream
ref: https://github.com/serde-rs/serde/pull/2421#pullrequestreview-1360210728

**Comment by augustuswm** [comment]
Yeah, it has it's own generator. I have a fix for this in the meantime and working on getting a new version of `octorust` published.

**Comment by ahl** [comment]
Ok, I would love to switch over to progenitor whenever we can and will do unspeakable things to make that happne...
